src/components/Dashboard/TodoList.vue

```diff
@@ -1498,6 +1498,29 @@ export default class TodoList extends Vue {
     }
   }
 
+  // navigate to Create UI if Full/Limited restoration or to Edit UI if Limited extension/Full to Limited conversion
+  buildRestorationUrl (item: TodoItemIF): string {
+    let url: string
+    let restorationType: string
+
+    if (item.filingSubType === RestorationTypes.FULL || item.filingSubType === RestorationTypes.LIMITED) {
+      url = `${this.getCreateUrl}?id=${this.getIdentifier}`
+    }
+
+    if (item.filingSubType === RestorationTypes.LTD_EXTEND || item.filingSubType === RestorationTypes.LTD_TO_FULL) {
+      if (item.filingSubType === RestorationTypes.LTD_EXTEND) {
+        restorationType = RestorationTypes.LTD_EXTEND
+      }
+
+      if (item.filingSubType === RestorationTypes.FULL) {
+        restorationType = RestorationTypes.FULL
+      }
+
+      url = `${this.getEditUrl}${this.getIdentifier}/` + restorationType + `?restoration-id=${item.filingId}`
+    }
+    return url
+  }
```

Comment:
Overall I'm OK with this.

Ideally, `buildRestorationUrl()` would look similar to `goToRestorationFiling()` (since they mostly do the same thing)... I think it would also look simpler to do the filingSubType check in `doResumeFiling()`.

What do you think? This isn't a deal-breaker.